seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/BrowserDriver.scala

```diff
@@ -0,0 +1,58 @@
+package org.scalajs.jsenv.selenium
+
+import org.openqa.selenium.remote._
+
+import org.scalajs.jsenv.JSConsole
+
+trait BrowserDriver {
```

Comment:
Could/should this be an `abstract class` instead? It would be more resilient wrt binary evolution.